nexus/src/external_api/params.rs

```diff
@@ -802,6 +804,18 @@ pub struct SshKeyCreate {
     pub public_key: String,
 }
 
+// METRICS
+
+/// Query parameters common to resource metrics endpoints.
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct ResourceMetrics {
```

Comment:
I think we need to clarify and document whether these are inclusive or exclusive of either end. That impacts how we'll use them in the actual query to ClickHouse.

Reply:
Docs added.